src/index.js

```diff
@@ -96,12 +97,31 @@ async function initApp() {
   // TODO: remove in next release
   app.use('/v1/nominatim', nominatimRouter);
   app.use('/api/v1/nominatim', nominatimRouter);
-  
-  process.on('SIGINT', function() {
-    logger.info( "\nGracefully shutting down from SIGINT (Ctrl-C)" );
-    // some other closing procedures go here
-    process.exit(0);
-  });
+
+  // Add headers so mapliber-gl decodes protobufs appropriately
+  const staticTilesOpts = { 
+    setHeaders: (res, _u1, _u2) => {
+      res.setHeader('Content-Encoding', 'gzip');
+      res.setHeader('Content-Type', 'application/x-protobuf');
+    },
+  };
+
+  // Expose static endpoint for tileset containing route-lines
+  const routeTiles = express.static(
+    path.join(WEB_APP_GEO_CONFIG_FOLDER_CONTAINER_PATH, 'route-tiles'),
+    staticTilesOpts
+  );
+  app.use('/v1/route-tiles', routeTiles);
+  app.use('/api/v1/route-tiles', routeTiles);
+  
+  // Expose static endpoint for tileset containing stop points
+  const stopTiles = express.static(
+    path.join(WEB_APP_GEO_CONFIG_FOLDER_CONTAINER_PATH, 'stop-tiles'),
+    staticTilesOpts
+  );
+  app.use('/v1/stop-tiles', stopTiles);
+  app.use('/api/v1/stop-tiles', stopTiles);
```

Comment:
@Andykmcc any thoughts on what else I can do to make these cache-friendly?
These tiles are very static and will only be updated nightly. However since `trip-ids` are encoded into them I need the cache to ideally invalidate with every nightly gtfs-refresh.

Reply:
set the cache-control header to 24 hours. You can see [here](https://github.com/search?q=repo%3Abikehopper%2Fbikehopper-web-app%20cache-control&type=code) how i do this. FWIW, all GET requests to web-app have a 12 hour cache-control header set, unless its overwritten. you can see in that link where i set the default in the `src/index.js`, then set custom controls for specific routes. You should follow that pattern. but honestly you should probably just do nothing and rely on the default 12 hour header.